docs/Configuration/yaml-files.md

```diff
@@ -330,6 +329,8 @@ software:
   # path is relative to default.yml, teams/team-name.yml, or teams/no-team.yml
   app_store_apps:
     - app_store_id: '1091189122'
+  fleet_maintained_apps:
+    - fleet_maintained_app_name: Microsoft Excel
```

Comment:
Dev note(s):
1. Users should be able to enter strings using whatever capitalization they prefer. In other words, this entry should be caps-agnostic.
2. If the string does not match an existing Fleet-maintained app, show an error. "[Entered name] is not a Fleet-maintained app. Please try again."